sympy/solvers/ode/tests/test_systems.py

```diff
@@ -1456,8 +1456,10 @@ def test_sysode_linear_neq_order1_type4():
             Eq(h(x), -C1/3 + 2*C2/3 - C3/3 + (C1/3 + C2/3 +
                 C3/3)*exp(x*(3*log(x) - 3)) + exp(x*(3*log(x) -
                     3))*Integral(exp(3*x)*exp(-3*x*log(x))*sin(x), x))]
-    with dotprodsimp(True):
-        assert dsolve(eqs7, simplify=False, doit=False) == sol7
+
+     # it will definately work after some updates in lambert (as this equation is in the form of lambert)
+     # with dotprodsimp(True):
+        # assert dsolve(eqs7, simplify=False, doit=False) == sol7
```

Comment:
We don't just disable tests like this unless there is a good reason why the test can't be made to work.

What is the input and output of solveset that causes this test to fail?

Reply:
The reason for this is that the lambert is not fully matured function this will work after the segregation of reals and complexes with domain

Reply:
This would be so much faster if you would answer my questions directly:

> What is the input and output of solveset that causes this test to fail?

Reply:
Actually this is already discussed here, in this comment https://github.com/sympy/sympy/pull/20537#issuecomment-739254138

Reply:
You've marked this as resolved but it is not resolved. This test should not be disabled.

Reply:
This will only work after the last PR/Commit.